react/PageBlock/PageBlock.js

```diff
@@ -0,0 +1,13 @@
+import styles from './PageBlock.less';
+import React from 'react';
+import classnames from 'classnames';
+
+export default function PageBlock({ children, className }) {
+  return (
+    <div className={classnames(className)}>
```

Comment:
I agree this is simpler, but was this resulting in `class="false"` on the element?